lib/matplotlib/axes/_axes.py

```diff
@@ -5761,6 +5762,8 @@ def hist(self, x, bins=10, range=None, normed=False, weights=None,
             for m, c in zip(n, color):
                 if bottom is None:
                     bottom = np.zeros(len(m), np.float)
+                if input_empty:
+                    width = 0
```

Comment:
I think the parenthetical widths comment was for drawing horizontal bars
and that the example with the line is correct.